framework/scenario/src/scenario/model/transaction/tx_response.rs

```diff
@@ -108,7 +110,10 @@ impl TxResponse {
                 return TxResponseStatus::signal_error(&error);
             }
 
-            let error_raw = base64::decode(topics.unwrap().get(1).unwrap()).unwrap();
+            let error_raw = general_purpose::STANDARD
```

Comment:
would be nice to extract this logic into a base64_decode function and place it somewhere in the crate root